linode/vpc/framework_schema_resource.go

```diff
@@ -0,0 +1,53 @@
+package vpc
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/int64planmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/planmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/stringplanmodifier"
+	"github.com/linode/terraform-provider-linode/linode/helper/customtypes"
+)
+
+var frameworkResourceSchema = schema.Schema{
+	Attributes: map[string]schema.Attribute{
+		"id": schema.Int64Attribute{
+			Description: "The id of the VPC.",
+			Computed:    true,
+			PlanModifiers: []planmodifier.Int64{
+				int64planmodifier.UseStateForUnknown(),
+			},
+		},
+		"label": schema.StringAttribute{
+			Description: "The label of the VPC. Only contains ascii letters, digits and dashes",
+			Required:    true,
+		},
+		"region": schema.StringAttribute{
+			Description: "The region of the VPC.",
+			Required:    true,
+			PlanModifiers: []planmodifier.String{
+				stringplanmodifier.RequiresReplace(),
+			},
+		},
+		"description": schema.StringAttribute{
+			Description: "The user-defined description of this VPC.",
+			Optional:    true,
+			Computed:    true,
+			PlanModifiers: []planmodifier.String{
+				stringplanmodifier.UseStateForUnknown(),
+			},
+		},
```

Comment:
```suggestion
			PlanModifiers: []planmodifier.String{
				stringplanmodifier.UseStateForUnknown(),
			},
			Default:     stringdefault.StaticString(""),
		},
```

Let's make it default to an empty string.

Reply:
I added `Computed` to this field, so this value can also be returned from the server.